adminpages/addons.php

```diff
@@ -410,16 +415,13 @@
 					var view_items = $(`[data-search-${view}]`);
 					var view_val = $(this).data('view');
 
-					// Update the URL hash.
-					$( this ).attr( 'href' ).replace( /#/, '' );

```

Comment:
Removed this line of code as it wasn't running anywhere and we used the `view_val` instead (retrieves the same value).